.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "semi": true,
-  "printWidth": 100,
```

Comment:
any specific reason for this change?

Reply:
the test coverage for [AuthenticatedRoute file](https://github.com/glific/glific-frontend/blob/0478c4104acedecd3b56cda618ccbcd247086a59/src/routes/AuthenticatedRoute/AuthenticatedRoute.tsx) was not covering all lines because it was going to the next line